xcode/Subconscious/Shared/Models/Slashlink.swift

```diff
@@ -46,7 +43,12 @@ public struct Slashlink:
     public var id: String { description }
     
     // The normalized markup form of the slashlink
-    public var markup: String { description }
+    public var markup: String {
+        guard let peer = peer else {
+            return slug.markup
+        }
+        return "\(peer.markup)\(slug.markup)"
+    }
```

Comment:
_Only_ `.markup` will strip the `local:subconscious:did` prefix, `description` preserves the old behaviour as-is.

Reply:
That's perfect. We want `description` to be `LosslessStringConvertible, so it should include the full did or petname.